### PR TITLE
add bind and dhcp

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

**Comment by yuwata** [comment]
Ah, I forgot to add headers...

File: recipes-debian/bind/bind_debian.bb [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+USERADD_PACKAGES = "${PN}"
+USERADD_PARAM_${PN} = "--system --home ${localstatedir}/cache/bind --no-create-home \
+                       --user-group bind"
+
+INITSCRIPT_NAME = "bind"
+INITSCRIPT_PARAMS = "defaults"
+
+SYSTEMD_SERVICE_${PN} = "named.service"
+
+do_sed_native_pkcs11() {
```
Comment (masami256) [inline recipes-debian/bind/bind_debian.bb]: @yuwata  Usually do_ prefix is used for task name. Thereby, if  shell/python function isn't a task, function name doesn't need to have do_ prefix.

Comment (yuwata) [reply recipes-debian/bind/bind_debian.bb]: OK. `do_` prefix is dropped. PTAL.

**Comment by masami256** [comment]
I did simple test that works okay!

dhcp

Add following setting in /etc/dhcp/dhcpd.conf and start dhcpd.

```
subnet 192.168.7.0 netmask 255.255.255.0 {                                                                                                                                                            
  option routers                192.168.7.1;                                                                                                                                                          
  option subnet-mask            255.255.255.0;                                                                                                                                                        
  option domain-name-servers    8.8.8.8;                                                                                                                                                              
  option time-offset            -18000;                                                                                                                                                               
  range 192.168.7.100 192.168.7.200;                                                                                                                                                                  
} 
```

Then run dhclient command to get ip address from server is succeeded.

```
root@qemuarm64:~# dhclient eth0
root@qemuarm64:~# ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether 52:54:00:12:34:02 brd ff:ff:ff:ff:ff:ff
    inet 192.168.7.100/24 brd 192.168.7.255 scope global eth0
       valid_lft forever preferred_lft forever
```

bind

```
root@qemuarm64:~# /etc/init.d/bind start
Starting domain name service: namedroot@qemuarm64:~# 
root@qemuarm64:~# /etc/init.d/bind status
Usage: /etc/init.d/bind {start|stop|reload|restart|force-reload}
root@qemuarm64:~# ps aex | grep bind
 1529 ?        Ssl    0:00 /usr/sbin/named -u bind SHELL=/bin/sh RUNLEVEL=5 PWD=/ runlevel=5 HOME=/ TERM=linux INIT_VERSION=sysvinit-2.93 SHLVL=2 CONSOLE=/dev/console PATH=/sbin:/bin:/usr/sbin:/usr/bin PREVLEVEL=N VERBOSE=no previous=N _=/sbin/start-stop-daemon
 2018 ttyAMA0  S+     0:00 grep bind SHELL=/bin/sh EDITOR=vi PWD=/home/root LOGNAME=root HOME=/home/root TERM=vt102 USER=root SHLVL=0 PS1=\u@\h:\w\$  HUSHLOGIN=FALSE PATH=/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin HZ=100 _=/bin/grep
```
dig

```
root@qemuarm64:~# !dig
dig +short  www.google.com
172.217.26.4
```

**Pull request merged**